src/app/scripts/cac/map/cac-map-control.js

```diff
@@ -471,10 +501,15 @@ CAC.Map.Control = (function ($, Handlebars, cartodb, L, turf, _) {
                                 map.removeLayer(lastItineraryHoverMarker);
                                 lastItineraryHoverMarker = null;
                                 dragging = false;
-                                startDragPoint = null;
+                                newWaypointIndex = null;
                             }
                         }, 3000);
-                    });
+                    }).on('drag', function(event) {
+                        // get itinerary from collection to pick up newer version with
+                        // layer modified by live dragging updates
+                        redrawWaypointDrag(event, newWaypointIndex, true);
+                    }).bindPopup('Drag marker to change route',
```

Comment:
No reason, apart from it having different syntax than the event bindings above. I'll move it above the event bindings.